pkg/api/grpcServer_test.go

```diff
@@ -459,7 +466,7 @@ func TestRudeClose(t *testing.T) {
 }
 
 func TestBrokenStream(t *testing.T) {
-	ts, err := prepTestState(30000)
+	ts, err := prepTestState(30009)
```

Comment:
I thought bazel sandboxed ports in tests? AFAIK you could run two http servers in different tests on the "same" port. I suppose we could write a quick test to find out.